Classes/MDMPersistenceController/MDMPersistenceController.m

```diff
@@ -85,19 +109,19 @@ - (NSPersistentStoreCoordinator *)setupNewPersistentStoreCoordinator {
                                              NSMigratePersistentStoresAutomaticallyOption:@YES
                                              };
     NSError *persistentStoreError;
-    NSPersistentStore *persistentStore = [persistentStoreCoordinator addPersistentStoreWithType:NSSQLiteStoreType
+    NSPersistentStore *persistentStore = [persistentStoreCoordinator addPersistentStoreWithType:storeType
                                                                                   configuration:nil
                                                                                             URL:self.storeURL
                                                                                         options:persistentStoreOptions
                                                                                           error:&persistentStoreError];
-    if (persistentStore == nil) {
+    if (persistentStore == nil && [storeType isEqualToString:NSSQLiteStoreType]) {
```

Comment:
What happens if the `persistentStore` is `nil` and the `storeType` is `NSInMemoryStoreType`?